scripts/test-binary.sh

```diff
@@ -14,6 +14,7 @@ function get_help {
     echo "  t | tags 'tags list'   Sets the tags to use"
     echo "  c | clear              Clears list of run files"
     echo "  p | pause              Pause after each stage"
+    echo "  n | names 'names list' List of file names to test [QUOTES NEEDED]"
```

Comment:
One more example below for using this feature would ease the understanding (although I know this was mentioned in doc).

Reply:
Examples added